github/ci/prow-deploy/kustom/base/kustomization.yaml

```diff
@@ -18,6 +18,8 @@ resources:
   - manifests/local/prow-phased-service.yaml
   - manifests/local/referee-deployment.yaml
   - manifests/local/referee-service.yaml
+  - manifests/local/github-jira-proxy-deployment.yaml
+  - manifests/local/github-jira-proxy-service.yaml
   - manifests/local/referee-servicemonitor.yaml
```

Comment:
Do we need an ingress like for gcsweb and deck? So that the webhook can hit the service?

Reply:
Daniel didn't mention it so i'm not sure, I tested it using kind cluster and manual post request `curl -X POST http://localhost:9900/ -H 'Content-Type: application/json' -H 'X-Hub-Signature-256:***...`
you think we need it in order to get external post requests?
  
If yes you mean create a new ingress, something like:
`
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt
  name: github-jira-proxy-ingress
  namespace: default
spec:
  tls:
  - hosts:
    - github-jira-proxy.example.com
    secretName: github-jira-proxy-tls
  rules:
  - host: github-jira-proxy.example.com 
    http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: github-jira-proxy-service
            port:
              number: 80
`

not sure what should be the host. also is cert-manager.io/cluster-issuer needed? if yes how I know the tls secret?

Reply:
yes I think you will need an ingress - we don't export any node ports or anything like that. It would be similar to the ingress for deck and gcsweb.

Reply:
Trying to implement, thanks

Reply:
@brianmcarey Added the file but not sure how to test if it's working, any ideas? thanks a lot